spec/lib/jobs/update-lookups-snmp-spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+var uuid = require('node-uuid');
+
+describe('update-lookups-job', function() {
```

Comment:
it's better to add some negative test cases:
- The catalog source 'snmp-1' doesn't exist.
- Fail to insert lookup to database.